<launch>

  <arg name="robot_name"      default="i2ws" />
  <arg name="global_frame_id" default="$(arg robot_name)/odom" />
  <arg name="file_name"       default="path_points" />
  
  <group ns="$(arg robot_name)">
    <node pkg="tuw_waypoint_to_spline" type="tuw_waypoint_to_spline_node" name="tuw_waypoint_to_spline_node" output="screen">
        <param name="global_frame" value="$(arg global_frame_id)" />
	    <rosparam command="load" file="$(find tuw_waypoint_to_spline)/cfg/$(arg file_name).yaml"/>
    </node>
  </group>
</launch>
